src/test/java/com/dlsc/jfxcentral/data/DataRepository2Test.java

```diff
@@ -693,6 +678,40 @@ public void shouldGetBookById() {
         });
     }
 
+    @Test
+    public void shouldGetLearnById() {
+        // given
+        DataRepository2 repository = DataRepository2.getInstance();
+        repository.reload();
+
+        assertFalse(repository.getLearn(LearnType.JAVA_FX).isEmpty());
+
+        // when
+        repository.getLearn(LearnType.JAVA_FX).forEach(learn -> {
+            Optional<Learn> result = repository.getLearnById(LearnType.JAVA_FX, learn.getId());
+
+            // then
+            assertTrue(result.isPresent(), "no learn returned for ID " + learn.getId());
+        });
+    }
+
+    @Test
+    public void shouldLoadLearnDescription() {
+        // given
+        DataRepository2 repository = DataRepository2.getInstance();
+        repository.reload();
+
+        assertFalse(repository.getLibraries().isEmpty());
+
+        // when
+        repository.getLearn(LearnType.JAVA_FX).forEach(lib -> {
+            String text = repository.getLearnReadMe(LearnType.JAVA_FX, lib);
+
+            // then
+            assertNotNull(text, "text missing for library ID " + lib.getId());
```

Comment:
copy paste error: not "library" but "learn".

Reply:
fixed